site/src/examples/link/OpenInANewTab.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactElement } from "react";
+import { StackLayout, Link } from "@salt-ds/core";
+import { GithubIcon } from "@salt-ds/icons";
+
+export const OpenInANewTab = (): ReactElement => (
+  <StackLayout>
+    <Link
+      href="https://www.saltdesignsystem.com"
+      target="_blank"
+      rel="noopener"
+    >
+      Opens in a new tab
+    </Link>
+    <Link
+      href="https://github.com/jpmorganchase/salt-ds"
+      target="_blank"
+      rel="noopener"
+      IconComponent={GithubIcon}
+    >
+      Salt GitHub repository
+    </Link>
+    <Link href="#" target="_blank" rel="noopener" IconComponent={null}>
+      This link has no icon
+    </Link>
```

Comment:
The links are a bit inconsistent here, one describes how the link will open, the next where it goes and the last what it looks like. What if put some descriptive text outside the link and made the actual link text describe where it goes (as per our guidance). For example:

```suggestion
 <p>
    Using default icon:
    {" "}
    <Link
      href="https://www.saltdesignsystem.com"
      target="_blank"
      rel="noopener"
    >
      Open the Salt website in a new tab
    </Link>
  </p>
  <p>
    Using a custom icon:
    {" "}
    <Link
      href="https://github.com/jpmorganchase/salt-ds"
      target="_blank"
      rel="noopener"
      IconComponent={GithubIcon}
    >
      Open the Salt GitHub repository in a new tab
    </Link>
  </p>
  <p>
    With the icon removed:
    {" "}
    <Link href="" target="_blank" rel="noopener" IconComponent={null}>
      Open this page in a new tab
    </Link>
```